awx/ui_next/CONTRIBUTING.md

```diff
@@ -320,3 +329,9 @@ You can learn more about the ways lingui and its React helpers at [this link](ht
 3) Open up the .po file for the language you want to test and add some translations.  In production we would pass this .po file off to the translation team.
 4) Once you've edited your .po file (or we've gotten a .po file back from the translation team) run `npm run compile-strings`.  This command takes the .po files and turns them into a minified JSON object and can be seen in the `messages.js` file in each locale directory.  These files get loaded at the App root level (see: App.jsx).
 5) Change the language in your browser and reload the page.  You should see your specified translations in place of English strings.
+
+### Marking an issue to be translated
+
+1) Issues marked with `component:I10n` should not be closed after the issue was fixed.
+2) Remove the label `state:needs_devel`.
+3) Add the label `state:pending_translations` .This will add the issue to the backlog of issues to be translated.
```

Comment:
It might be good to clarify step 3 by adding, "At this point, the translations will be batch translated by a maintainer, creating relevant entries in the PO files.  Then after those translations have been merged, the issue can be closed."